UPGRADING.md

```diff
@@ -315,6 +315,31 @@ message MsgSetWithdrawAddress {
 }
 ```
 
+#### `{accepts,implements}_interface` proto annotations
```

Comment:
This should be under a new section in `## 0.47` instead of 0.46

Reply:
+1

Reply:
fixed